pom.xml

```diff
@@ -81,6 +81,8 @@
 		<package-name>net.haesleinhuepf</package-name>
 		<license.licenseName>bsd_3</license.licenseName>
 		<license.copyrightOwners>Robert Haase, MPI CBG</license.copyrightOwners>
+		<imagej.app.directory>C:/programs/fiji-win64/Fiji.app/</imagej.app.directory>
+		<!--<imagej.app.directory>/home/rhaase/programs/fiji/Fiji.app/</imagej.app.directory>-->
```

Comment:
This should not be defined here. Use `~/.m2/settings.xml` instead.

Reply:
Ah cool. Thanks, @ctrueden! Didn't know about that :-)

Reply:
:+1: It's nice because you can set it to match your Fiji install on each individual system. Sorry I wasn't clear about my rationale for removing it before.